bin/ci/ubuntu/build-and-test.sh

```diff
@@ -66,11 +66,13 @@ fi
 # if we get a crash.
 gdb -return-child-result -quiet -batch \
     -ex "set env MALLOC_CHECK_=3" \
+    -ex "set demangle-style none" \
     -ex "set print thread-events off" \
+    -ex "file $APP_PATH" \
+    -ex "set args $APP_ARGS" \
     -ex run \
     -ex "thread apply all backtrace full" \
     -ex "quit" \
-    --args $APP_PATH $APP_ARGS
```

Comment:
not really - we run the `rippled` unit tests under `gdb` in travis, presumably to gather a stack track if there is a crash.  `gdb` tries to demangle symbols when it loads the binary, and it's crashing on these symbols now. This is a `gdb` bug -- an alternative solution would be to update `gdb` on the Travis setup, assuming this is indeed fixed in a current release. I think we wanted to first confirm that demangling was the source of the CI crash with this change, then we can decide whether to try upgrading the tools or just skip the demangling like this.